state/execution.go

```diff
@@ -705,6 +755,11 @@ func fireEvents(
 	}
 
 	for i, tx := range block.Data.Txs {
+		//TODO:  can we decode without decoding the whole blob?
```

Comment:
is this a residual comment or a question?

Reply:
question, for later on but can be ignored now. ill remove